app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -104,12 +104,22 @@ const StoryLabels = ({ labels }) => {
   );
 };
 
-const StoryInfo = ({ storyType ,title, labels, estimate }) => (
+export const StoryTitle = ({title, ownedByInitials, ownedByName}) => (
```

Comment:
You should keep the space inside of curly braces
```suggestion
export const StoryTitle = ({ title, ownedByInitials, ownedByName }) => (
```